packages/js/github-actions/actions/get-plugin-releases/README.md

```diff
@@ -0,0 +1,64 @@
+# Get latest releases versions from a specific plugin or from wordpress core
```

Comment:
This action currently only supports plugins that have been published to https://wordpress.org/. Other plugins, for example, the AutomateWoo plugin, are not supported.

It's recommended to describe the supported scope here, and perhaps also the `slug` parameter in action.yml.